contracts/crosschain/axelar/AxelarGatewayDestination.sol

```diff
@@ -53,6 +84,7 @@ abstract contract AxelarGatewayDestination is AxelarGatewayBase, AxelarExecutabl
         );
 
         bytes4 result = IERC7786Receiver(receiver.parseAddress()).executeMessage(
+            messageId,
```

Comment:
Here we have a choice between:
- the axelar adaptor passes the axelar commandId to the receiver (stringified)
- the axelar adaptor passes an empty string (and hides the axelar adaptor)

Note that the AxelarExecutor (that is part of  `@axelar-network/axelar-gmp-sdk-solidity`) is implement in a way that doesn't pass the commandId to the "execution logic". If axelar believes its not necessary for execution, maybe we can accept that the adaptor doesn't forward it further and passes an empty string instead.

Reply:
If the id exists we should forward it.

Reply:
> Note that the AxelarExecutor (that is part of `@axelar-network/axelar-gmp-sdk-solidity`) is implement in a way that doesn't pass the commandId to the "execution logic".

Apparently this is only true for an older version of `AxelarExecutable`. Their v6 does pass the command id. We should just update the dep and keep using that.

https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/v6.0.6/contracts/executable/AxelarExecutable.sol#L60-L61